### PR TITLE
LPS-169652 The Pages collapsable button is collapsed when navigate to the Page Tree through admin

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
Hey @veroglez! As discussed in Slack, I had added the root layout id in backend, so I think it's better to fix the bug there 😄

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/java/com/liferay/product/navigation/product/menu/web/internal/display/context/LayoutsTreeDisplayContext.java [hunk 1]
```diff
@@ -700,15 +700,18 @@ private String _getRedirect() {
 	private List<Long> _getSelectedLayoutPath() throws Exception {
 		long selPlid = getSelPlid();
 
+		List<Long> selectedLayoutPath = new ArrayList<>();
```
Comment (jkappler) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/java/com/liferay/product/navigation/product/menu/web/internal/display/context/LayoutsTreeDisplayContext.java:703]: @veroglez, we can initialize the List directly, and also use the PLID accesor, like this:

```
private List<Long> _getSelectedLayoutPath() throws Exception {
		long selPlid = getSelPlid();

		List<Long> selectedLayoutPath = Arrays.asList(
			LayoutConstants.DEFAULT_PARENT_LAYOUT_ID);

		if (selPlid <= 0) {
			return selectedLayoutPath;
		}

		selectedLayoutPath.addAll(
			ListUtil.toList(
				_layoutService.getAncestorLayouts(selPlid),
				Layout.PLID_ACCESSOR));

		return selectedLayoutPath;
	} 
```

Comment (jkappler) [reply modules/apps/product-navigation/product-navigation-product-menu-web/src/main/java/com/liferay/product/navigation/product/menu/web/internal/display/context/LayoutsTreeDisplayContext.java:703]: Talking to @lfbesada, maybe the accesor is not a good fit here, but we could use:

```
selectedLayoutPath.addAll(
			ListUtil.toList(
				_layoutService.getAncestorLayouts(selPlid), Layout::getPlid)); 
```

**Review by jkappler — APPROVED** [review]

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**